src/Symfony/Component/Validator/Constraints/Email.php

```diff
@@ -49,11 +49,19 @@ class Email extends Constraint
     );
 
     public $message = 'This value is not a valid email address.';
+
+    /**
+     * @deprecated since Symfony 4.2.
```

Comment:
the `*_CHECK_FAILED_ERROR` consts should also be marked `@deprecated`

Reply:
done